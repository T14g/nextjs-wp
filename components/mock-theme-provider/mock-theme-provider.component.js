import React from "react";
import 'jest-styled-components'
import { ThemeProvider } from "styled-components";
import Theme from "../../themes/default";

export const MockThemeProvider = ({ children }) => (
    <ThemeProvider theme={Theme}>{children}</ThemeProvider>
)