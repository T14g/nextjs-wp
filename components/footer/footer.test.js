import React from "react";
import renderer from 'react-test-renderer';
import Footer from "./footer.component";
import { MockThemeProvider } from "../mock-theme-provider/mock-theme-provider.component";

it('Render Footer correctly', () => {
    const tree = renderer.create(
        <MockThemeProvider>
            <Footer />
        </MockThemeProvider>
    ).toJSON();

    expect(tree).toMatchSnapshot();
});
