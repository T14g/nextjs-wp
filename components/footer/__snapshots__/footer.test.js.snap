// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Footer correctly 1`] = `
<footer
  className=""
>
  Â© 2021 T14g All Rights Reserved
</footer>
`;
